src/legacy/server/saved_objects/lib/import.ts

```diff
@@ -134,18 +141,13 @@ export function createObjectsFilter(
     ) {
       return true;
     }
-    let hasReferenceToReplace = false;
+    const refReplacements = replaceReferences.map(ref => `${ref.type}:${ref.from}`);
```

Comment:
I'd prefer if we did this outside the filter function so it was reused for each call to `filter(obj)`